### ISSUE TITLE
Unreasonablely slow booting caused on RPI 3B+

### ISSUE THREAD

**Issue by tomty89** [gonzoua/rpi3-psci-monitor#5]
I am having issue with this when I try to boot FreeBSD on an Raspberry Pi 3B+. Booting is intolerably slow as long as this (armstub8.bin) exists in the FAT-formatted boot partition. The slowness even shows in timeout count downs:

https://drive.google.com/file/d/1imGnq-bUFx5zbnkEoB5ZJNC4qnv7OvDx/view?usp=drivesdk

I don't know how much FreeBSD is supposed to rely on this but once I removed it, the kernel or something panics once I try to boot it. ~~(I can upload a photo of it later if necessary.)~~

![DSC_0020](https://user-images.githubusercontent.com/8604057/71736402-ec6b2c80-2e8b-11ea-9fec-2122b4106e0d.JPG)

Also, not sure if it is related to this, but I can't have any USB devices detected when the kernel kicks in (mouse/keyboard, thumb drive, onboard NIC). They work during the "slow stage" though (I can press enter to skip the timeout).

**Comment by tomty89** [comment]
hexdump difference of a fine build and a problematic build:
```
[tom@alarm ~]$ diff <(hexdump -C 1/armstub8.bin) <(hexdump -C armstub8.bin)
1c1
< 00000000  00 00 a8 d2 1f 00 00 b9  01 00 b0 52 01 08 00 b9  |...........R....|
---
> 00000000  00 f0 bf d2 1f 00 00 b9  01 00 b0 52 01 08 00 b9  |...........R....|
12c12
< 000000b0  00 f8 24 01 00 00 00 00  30 08 c5 30 00 00 00 00  |..$.....0..0....|
---
> 000000b0  80 f9 37 03 00 00 00 00  30 08 c5 30 00 00 00 00  |..7.....0..0....|
[tom@alarm ~]$
```
They are supposed to be build from the same commit (2b8890a). The former is extracted from the 12.1-RELEASE RPI3 image. The latter provided by @kevans91.

**Comment by tomty89** [comment]
Fixed with #6.
